docs/index.md

```diff
@@ -166,6 +166,10 @@ For example:
 - `github.com/turbot/steampipe-plugin-aws//**/*.tf?ref=fix_7677` matches all Terraform configuration files in the specific tag of a repository.
 - `github.com/turbot/steampipe-plugin-aws//aws-test/tests/aws_acm_certificate//*.tf` matches all Terraform configuration files in the specified folder path.
 
+If the above specified format does not work please try the below format:
+
+- `git::ssh://git@github.com/test_org/test_repo//*.tf`
```

Comment:
```suggestion
If the example formats above don't work for private repositories, this could be due to git credentials being stored by another tool, e.g., VS Code. An alternative format you can try is:

- `git::ssh://git@github.com/test_org/test_repo//*.tf`
```

Reply:
updated